data/autoyast_sle15/autoyast_systemd_timesync.xml

```diff
@@ -0,0 +1,89 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE profile>
+<profile xmlns="http://www.suse.com/1.0/yast2ns" xmlns:config="http://www.suse.com/1.0/configns">
+    <!-- minimal autoyast profile -->
+    <suse_register>
+      <do_registration config:type="boolean">true</do_registration>
+      <email/>
+      <reg_code>{{SCC_REGCODE}}</reg_code>
+      <install_updates config:type="boolean">true</install_updates>
+      <reg_server>{{SCC_URL}}</reg_server>
+      <addons config:type="list">
+        <addon>
+         <name>sle-module-basesystem</name>
+         <version>{{VERSION}}</version>
+         <arch>{{ARCH}}</arch>
+        </addon>
+      </addons>
+    </suse_register>
+        <general>
+        <mode>
+            <confirm config:type="boolean">false</confirm>
+        </mode>
+    </general>
+    <bootloader>
+        <global>
+            <timeout config:type="integer">-1</timeout>
+        </global>
+    </bootloader>
+    <networking>
+        <keep_install_network config:type="boolean">true</keep_install_network>
+    </networking>
+      <ntp-client>
+    <ntp_policy>auto</ntp_policy>
```

Comment:
could you fix the xml indentation?

Reply:
Hi @b10n1k  this change is not necessary, it is the same situation than with the yaml, you can find the right tool to do it and make a proposal with corresponding progress ticket, otherwise is ok accept anyone preference to indent files. Please let's discuss this point offline if not clear yet for you.

Reply:
yup. let do it offline. Can you resolve this because the option is not available to me

Reply:
ahm, I didn't know that, I will resolve it, thanks.